app/assets/javascripts/components/common/notifications.cjsx

```diff
@@ -1,30 +1,52 @@
 # Requirements
 #----------------------------------------
 
-React = require 'react'
+React             = require 'react'
+NotificationStore = require '../../stores/notification_store'
+NotificationActions = require '../../actions/notification_actions'
 
 
 # Component
 #----------------------------------------
 
 Notifications = React.createClass(
+
+  mixins: [NotificationStore.mixin]
+
+  getInitialState: ->
+    notifications: NotificationStore.getNotifications()
+
+  storeDidChange: ->
+    @setState
+      notifications: NotificationStore.getNotifications()
+
+  _handleClose: (notification) ->
+    NotificationActions.removeNotification(notification)
+
+  renderNotification: (notification, i) ->
```

Comment:
I like your convention of underscoring method names that aren't part of the lifecycle, like `_handleClose` above. Why not for `renderNotification`?